### DISCUSSION TITLE
Demo Scispacy model giving different results when compared to the latest version

### DISCUSSION THREAD

**Discussion by Yaz-lab** [explosion/spaCy#12907] [Help: Installation] [answered]
The spacy special model en_ner_bc5dr_md for chemical entity recognition however the results are different compared to the demo version. I am using the latest model however the older version seems to be performing well. can you please help me on what version does spacy use in their demo.

**Comment by rmitsch** [comment, marked as answer]
Hi @Yaz-lab, `scispacy` is a third-party package and isn't maintained by the spaCy team. I recommend you open an issue in the [`scispacy` repo](https://github.com/allenai/scispacy).

**Reply by Yaz-lab** [reply]
Thank you so much.
